bigchaindb/backend/localmongodb/query.py

```diff
@@ -228,7 +228,9 @@ def store_unspent_outputs(conn, *unspent_outputs):
                 )
             )
         except DuplicateKeyError:
-            # TODO log warning at least
+            raise DuplicateKeyError(
```

Comment:
The UTXO implementation is incomplete and currently [disabled](https://github.com/bigchaindb/bigchaindb/commit/8c32ae798bfbad2c580035a8b3bfc9e1959d56d0). There does not seem to be a need to change anything here.